frontend/src/routes/Governance/discovered/grouping.ts

```diff
@@ -142,14 +147,30 @@ export function grouping(): {
     const parseStringMap = eval(parseStringMapStr) as TParseStringMap
     const parseDiscoveredPolicies = eval(parseDiscoveredPoliciesStr) as TParseDiscoveredPolicies
 
+    const kyvernoViolationMap: { [policyNamespaceName: string]: number } = {}
+
+    if (kyvernoPolicyReports.length > 0) {
+      kyvernoPolicyReports.forEach((cr) => {
+        const kindNameViolation = cr.policyViolationCounts?.split('=') ?? []
+        if (kindNameViolation.length === 2) {
+          if (typeof kyvernoViolationMap[kindNameViolation[0]] === 'number') {
```

Comment:
Curious if this also works:
```suggestion
          if (kyvernoViolationMap[kindNameViolation[0]]) {
```

Reply:
What if violation is 0? O goes to else so